openbook_auth/models.py

```diff
@@ -2691,23 +2717,49 @@ def translate_post_comment_with_id(self, post_comment_id):
         )
         return post_comment, result.get('translated_text')
 
+    def subscribe_to_user_with_username(self, username):
```

Comment:
`def subscribe_to_user_with_username_notificactions` please

Reply:
better to say `subscribe_to_notifications_for_user_with_username`